gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -78,12 +81,13 @@
     // TODO: Clear feature markers when project is deactivated.
     // TODO: Since we depend on project stream from repo anyway, this transformation can be moved
     // into the repo?
-    this.features =
+    this.mapPins =
         LiveDataReactiveStreams.fromPublisher(
             projectRepository
                 .getActiveProjectOnceAndStream()
                 .map(Loadable::value)
-                .switchMap(this::getFeaturesStream));
+                .switchMap(this::getFeaturesStream)
+                .map(MapContainerViewModel::toMapPins));
```

Comment:
I'm guessing we can't use `this` here because it's a static method? gtk!

Reply:
Right! It also makes it clear that a reference to the current instance is not passed to the method impliciltly